core/templates/services/playthrough.service.ts

```diff
@@ -128,11 +139,16 @@ class EarlyQuitTracker {
     this.expDurationInSecs = expDurationInSecs;
   }
 
-  generateIssueCustomizationArgs(): EarlyQuitCustomizationArgs {
-    return {
-      state_name: {value: this.stateName},
-      time_spent_in_exp_in_msecs: {value: this.expDurationInSecs * 1000},
-    };
+  // Return undefined if no issue has been found.
```

Comment:
Not clear what you mean by "no issue has been found". See above as well.

Reply:
Done